components/keyrings/keyring_file/keyring_file.cc

```diff
@@ -183,10 +185,14 @@ bool set_paths(const char *component_path, const char *instance_path) {
                   Existing data remains as it is.
 */
 bool init_or_reinit_keyring() {
+#ifndef XTRABACKUP
   /* Get config */
   std::unique_ptr<Config_pod> new_config_pod;
   if (keyring_file::config::find_and_read_config_file(new_config_pod))
     return true;
+#else
+  std::unique_ptr<Config_pod> new_config_pod = std::move(xtrabackup::components::new_config_pod);
```

Comment:
@rahulmalik87 fixed the 80 characters and MacOS compilation issue. Added '-undefined dynamic_lookup' in case of MacOS to tell the linker that dynamic symbols will be looked at runtime instead of compilation time.